modules/ipi-install-diagnosing-duplicate-mac-address.adoc

```diff
@@ -0,0 +1,51 @@
+[id="ipi-install-diagnosing-duplicate-mac-address_{context}"]
+= Diagnosing a duplicate MAC address when provisioning a new host in the cluster
+
+If the MAC address of an existing bare-metal node in the cluster matches the MAC address of a bare-metal host you are attempting to add to the cluster, the Bare Metal Operator associates the host with the existing node. If the host enrollment, inspection, cleaning, or other Ironic steps fail, the Bare Metal Operator retries the installation continuously. A registration error is displayed for the failed bare-metal host.
+
+You can diagnose a duplicate MAC address by examining the bare-metal hosts that are running in the `openshift-machine-api` namespace.
+
+.Prerequisites
+
+* Install an {product-title} cluster on bare metal.
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+To determine whether a bare-metal host that fails provisioning has the same MAC address as an existing node, do the following:
+
+. Get the bare-metal hosts running in the `openshift-machine-api` namespace:
++
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
+NAME                 STATUS   PROVISIONING STATUS      CONSUMER
+openshift-master-0   OK       externally provisioned   openshift-zpwpq-master-0
+openshift-master-1   OK       externally provisioned   openshift-zpwpq-master-1
+openshift-master-2   OK       externally provisioned   openshift-zpwpq-master-2
+openshift-worker-0   OK       provisioned              openshift-zpwpq-worker-0-lv84n
+openshift-worker-1   OK       provisioned              openshift-zpwpq-worker-0-zd8lm
+openshift-worker-2   error    registering
+----
+
+. To see more detailed information about the status of the failing host, run the following command, and search for the failing host in the output YAML:
++
+[source,terminal]
+----
+$ oc get bmh -o yaml -n openshift-machine-api
```

Comment:
+100 on using `bmh` instead of pods. I don't want to spoil the fun, but I might have gone with:

`oc get -n openshift-machine-api bmh <bare-metal-host-name> -o yaml`

That said, I'm OK merging without it because I'm sure customers can work that one out for themselves.

Reply:
Added this! Procedure reads much better now. Thanks Mike! really appreciate it. LMK if you spot anything else.